msal/src/main/java/com/microsoft/identity/client/ITokenShare.java

```diff
@@ -34,12 +34,24 @@
      * {@inheritDoc}
      */
     @Override
-    String getWrappedFamilyRefreshToken(String identifier) throws MsalClientException;
+    String getOrgIdFamilyRefreshToken(String identifier) throws MsalClientException;
 
     /**
      * {@inheritDoc}
      */
     @Override
-    void saveFamilyRefreshToken(String tokenCacheItemJson) throws MsalClientException;
+    void saveOrgIdFamilyRefreshToken(String ssoStateSerializerBlob) throws MsalClientException;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    String getMsaFamilyRefreshToken(String identifier) throws MsalClientException;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    void saveMsaFamilyRefreshToken(String refreshToken) throws MsalClientException;
```

Comment:
Do we plan to add the support to remove the frt when removing account?

Reply:
For the _current_ app, `removeAccount()` will clear the FRT from the app local cache.

Do you mean will we remove the FRT from _other_ apps we might have shared the token with through TSL? I'm not aware of any such requirement; for apps not integrated with broker I wouldn't expect such functionality to exist.

Lmk if that answers your question 😄

Reply:
Synced offline for more context.

Basically, 'What should happen if I have two apps on the device integrated with TSL? `AppA` could share token with `AppB`. `AppA` removes account, `AppB` shares it back thereby not _really_ removing the account."

Brian's answer:
    Not sure what's expected here. Perhaps a new API is needed to signal to TSL 'remove tokens for any user with this id'. This method would need to accept an OID or a upn

Reply:
Let's discuss this internally.